lib/board.rb

```diff
@@ -39,11 +39,14 @@ def empty?(spot)
   end
 
   def create_board(board_size)
+    id = 0
+
     new_board = Array.new
     board_size.times do
       new_row = Array.new
       board_size.times do
-        new_row.push(Mark::EMPTY)
+        new_row.push(id)
```

Comment:
That's a good start, and you're right. That you want to display the numbers 0-8 (or 1-9) for a user is a UI concern, not a board's one.

Well...nil does have it's problems. But have a think what's more suitable to define the absence of a value and we'll discuss it on Friday.